README.md

```diff
@@ -1,2 +1,69 @@
-# rustls-symcrypt
-Glue between Rustls and rust-symcrypt 
+# SymCrypt Provider for Rustls
+
+This crate provides integration for using [rust-symcrypt](https://github.com/microsoft/rust-symcrypt) cryptographic functionalities with  [rustls](https://github.com/rustls/rustls), by implementing the required traits specified by `rustls`.
+
+## Status and Platform Support:
+- Currently works with: `rustls = { version = "0.23.0", features = ["ring", "tls12", "std"], default-features = false }`
+- Windows AMD64: Full support.
+- Linux Mariner: Full support.
+- Ubuntu: Partial support. While tested, full compatibility and optimal performance on all Ubuntu environments cannot be guaranteed.
```

Comment:
>Partial support. [](http://example.com/codeflow?start=10&length=16)

I'd say SymCrypt is fully supported on Ubuntu. But if you don't want to make that statement for the crate yet, that's fine.

Reply:
I'll leave it for now and update once Max has finished the Debian package